include/psp2/dmac5.h

```diff
@@ -0,0 +1,120 @@
+#define SCE_SBL_DMAC5_COMMAND_AES_256_ECB_DEC      (SCE_SBL_DMAC5_COMMAND_AES_ECB_DEC | SCE_SBL_DMAC5_COMMAND_AES_BIT_SIZE_256)
+
+#define SCE_SBL_DMAC5_COMMAND_AES_CBC              (SCE_SBL_DMAC5_COMMAND_BASIC_0x8)
+#define SCE_SBL_DMAC5_COMMAND_AES_CBC_ENC          (SCE_SBL_DMAC5_COMMAND_AES_CBC | SCE_SBL_DMAC5_COMMAND_AES_ECB_ENC)
+#define SCE_SBL_DMAC5_COMMAND_AES_CBC_DEC          (SCE_SBL_DMAC5_COMMAND_AES_CBC | SCE_SBL_DMAC5_COMMAND_AES_ECB_DEC)
+#define SCE_SBL_DMAC5_COMMAND_AES_128_CBC_ENC      (SCE_SBL_DMAC5_COMMAND_AES_CBC_ENC | SCE_SBL_DMAC5_COMMAND_AES_BIT_SIZE_128)
+#define SCE_SBL_DMAC5_COMMAND_AES_128_CBC_DEC      (SCE_SBL_DMAC5_COMMAND_AES_CBC_DEC | SCE_SBL_DMAC5_COMMAND_AES_BIT_SIZE_128)
+#define SCE_SBL_DMAC5_COMMAND_AES_192_CBC_ENC      (SCE_SBL_DMAC5_COMMAND_AES_CBC_ENC | SCE_SBL_DMAC5_COMMAND_AES_BIT_SIZE_192)
+#define SCE_SBL_DMAC5_COMMAND_AES_192_CBC_DEC      (SCE_SBL_DMAC5_COMMAND_AES_CBC_DEC | SCE_SBL_DMAC5_COMMAND_AES_BIT_SIZE_192)
+#define SCE_SBL_DMAC5_COMMAND_AES_256_CBC_ENC      (SCE_SBL_DMAC5_COMMAND_AES_CBC_ENC | SCE_SBL_DMAC5_COMMAND_AES_BIT_SIZE_256)
+#define SCE_SBL_DMAC5_COMMAND_AES_256_CBC_DEC      (SCE_SBL_DMAC5_COMMAND_AES_CBC_DEC | SCE_SBL_DMAC5_COMMAND_AES_BIT_SIZE_256)
+
+#define SCE_SBL_DMAC5_COMMAND_AES_CTR              (SCE_SBL_DMAC5_COMMAND_BASIC_0x10)
+#define SCE_SBL_DMAC5_COMMAND_AES_CTR_ENC          (SCE_SBL_DMAC5_COMMAND_AES_CTR | SCE_SBL_DMAC5_COMMAND_AES_ECB_ENC)
+#define SCE_SBL_DMAC5_COMMAND_AES_CTR_DEC          (SCE_SBL_DMAC5_COMMAND_AES_CTR | SCE_SBL_DMAC5_COMMAND_AES_ECB_DEC)
+#define SCE_SBL_DMAC5_COMMAND_AES_128_CTR_ENC      (SCE_SBL_DMAC5_COMMAND_AES_CTR_ENC | SCE_SBL_DMAC5_COMMAND_AES_BIT_SIZE_128)
+#define SCE_SBL_DMAC5_COMMAND_AES_128_CTR_DEC      (SCE_SBL_DMAC5_COMMAND_AES_CTR_DEC | SCE_SBL_DMAC5_COMMAND_AES_BIT_SIZE_128)
+#define SCE_SBL_DMAC5_COMMAND_AES_192_CTR_ENC      (SCE_SBL_DMAC5_COMMAND_AES_CTR_ENC | SCE_SBL_DMAC5_COMMAND_AES_BIT_SIZE_192)
+#define SCE_SBL_DMAC5_COMMAND_AES_192_CTR_DEC      (SCE_SBL_DMAC5_COMMAND_AES_CTR_DEC | SCE_SBL_DMAC5_COMMAND_AES_BIT_SIZE_192)
+#define SCE_SBL_DMAC5_COMMAND_AES_256_CTR_ENC      (SCE_SBL_DMAC5_COMMAND_AES_CTR_ENC | SCE_SBL_DMAC5_COMMAND_AES_BIT_SIZE_256)
+#define SCE_SBL_DMAC5_COMMAND_AES_256_CTR_DEC      (SCE_SBL_DMAC5_COMMAND_AES_CTR_DEC | SCE_SBL_DMAC5_COMMAND_AES_BIT_SIZE_256)
+
+#define SCE_SBL_DMAC5_COMMAND_AES_CMAC             (SCE_SBL_DMAC5_COMMAND_BASIC_0x1 | SCE_SBL_DMAC5_COMMAND_BASIC_0x2 | SCE_SBL_DMAC5_COMMAND_BASIC_0x8 | SCE_SBL_DMAC5_COMMAND_BASIC_0x10 | SCE_SBL_DMAC5_COMMAND_BASIC_0x20)
+#define SCE_SBL_DMAC5_COMMAND_AES_128_CMAC         (SCE_SBL_DMAC5_COMMAND_AES_CMAC | SCE_SBL_DMAC5_COMMAND_AES_BIT_SIZE_128)
+#define SCE_SBL_DMAC5_COMMAND_AES_192_CMAC         (SCE_SBL_DMAC5_COMMAND_AES_CMAC | SCE_SBL_DMAC5_COMMAND_AES_BIT_SIZE_192)
+#define SCE_SBL_DMAC5_COMMAND_AES_256_CMAC         (SCE_SBL_DMAC5_COMMAND_AES_CMAC | SCE_SBL_DMAC5_COMMAND_AES_BIT_SIZE_256)
+
+typedef struct SceDmac5EncdecParam {
```

Comment:
could you change it to
```suggestion
typedef struct SceDmac5EncDecParam {
```
?